README.md

```diff
@@ -1,19 +1,55 @@
 
-* See the [Editing Taskforce Charter](CHARTER.md)
+Details can be found in the [Group Charter](https://github.com/w3c/editing/blob/gh-pages/CHARTER.md).
 
-* See the [Selection API](https://w3c.github.io/selection-api/) specification, which defines the `Selection` interface, the `getSelection()` method, and the `selectstart` and `selectionchange` events.
+## Actively developed specs
 
+* [Content Editable Disabled](https://github.com/w3c/editing/blob/gh-pages/Active%20Documents/contentEditableDisabled.html)
+* [Virtual Keyboard Policy ](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/VirtualKeyboardPolicy/explainer.md)
+* [Selection API](https://w3c.github.io/selection-api/)
 
-**These specs are incomplete and it is not expected that they will advance beyond draft status:**
+## Graduated
+[Highlight API](https://drafts.csswg.org/css-highlight-api-1/)
+
+## Backlog
+* [EditContextAPI](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/EditContext/explainer.md)
+
+## Archive
 
 * [Content Editable True](http://w3c.github.io/editing/contentEditableTrue.html)
 
 * [execCommand](http://w3c.github.io/editing/execCommand.html)
 
+## Process
+
+### Participation
+Participation from ALL who are passionate about writing text editors on the web or have a question/suggestion on the topic is welcomed!
+
+There are couple of things you should know before you proceed.
+
+In accordance with W3C contribution requirements, if you intend to create PRs and/or spend a significant amount of time on these discussions, you should follow [Web Apps WG contribution guidance](https://github.com/w3c/editing/blob/gh-pages/CONTRIBUTING.md).
+
+
+### Meetings
+[Recurring Meeting Invite](https://calendar.google.com/event?action=TEMPLATE&tmeid=MDRhYWhjY2NjdnE0Y2RyY2l2N2oybTdnZmVfMjAyMDAzMTNUMTYwMDAwWiBnbHl1a0BtaWNyb3NvZnQuY29t&tmsrc=glyuk%40microsoft.com&scp=ALL)
+
+
+### Details:
+Editing TF meets remotely at 9:00AM PST, on second Friday of every month.
+
+Currently planned dates: April 10th, 2020 | May 8th, 2020 | etc.
+
+Meeting [agenda query](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fw3c%2Fediting%2Flabels%2FAgenda%252B&data=02%7C01%7CGrisha.Lyukshin%40microsoft.com%7C1c13e99db18e4385fa7808d7aecb5856%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637170061429746119&sdata=%2FlrhPGBZEnldazXwcv6ZwY%2BderREvECHD6cSKeuuIUY%3D&reserved=0)
+
+[Meeting Invite](https://calendar.google.com/event?action=TEMPLATE&tmeid=MDRhYWhjY2NjdnE0Y2RyY2l2N2oybTdnZmVfMjAyMDAzMTNUMTYwMDAwWiBnbHl1a0BtaWNyb3NvZnQuY29t&tmsrc=glyuk%40microsoft.com&scp=ALL)
+
+IRC:
+[irc.w3.org](https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Firc.w3.org%2F&data=02%7C01%7CGrisha.Lyukshin%40microsoft.com%7C1c13e99db18e4385fa7808d7aecb5856%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637170061429766112&sdata=Ar3vWCkYOxfk1j3vX4wLiui%2FiTqsuzlVXi1UI8wF0lY%3D&reserved=0)
```

Comment:
Kinda sucks that this link doesn't just go to https://irc.w3.org

Reply:
safe links...
I'll fix it in the next commit.